.travis.yml

```diff
@@ -40,3 +41,16 @@ script:
     if [ "$TRAVIS_TAG" == "1.0-stable" ]; then
       docker run -it f5devcentral/containthedocs publish-product-docs-to-prod connectors/k8s-bigip-ctlr v1.0
     fi
+
+notifications:
+  slack:
+    secure: Nk+RIhWd5KTiJj/s//bqNCcO6gJY/9/68ohN1fRSRCdIYNa0Pwu37SVdM3p1vwEJcFwyPLSFBcnFrY8raN7Ih12md3leP/ac6XDPx99PTAI9p5oKpREgQ4A7fE53rp0uGDEREp2ZFLW4O33hty3eP39wswpebAX6l8sdiE3YMaBTY6t2cakeesGsoOQck0IpNKTLWVq36+vVSKPueec510kjJPc0XPWLAEhoG7kU7IbYHx3250BO/k2l8Hw8ITlwft3E95y9gpwhlB/wia/I638S7QSTuheJmu5yyhrby8g+6HTyiUjjZB9ld3SjsM46QAm9T0ft2Cwx5JXJJZhYq07sWIiIVOimwXFQzEC9E9df+wWgwA2EglWstRn4LxUYgG40Jfl8BTrve9Zd+hYKJ8U9RzsYZK/gOoWzECgJ+U+i5s2DC+SMUpvP+skrH1grLXYuBLVVw3sWTHqCIUZLzunuhXhER1ogHMrO/o4mXmw1h0k+2fcKwbT95DTnP4a20n22uJ+Ox6dGpGSn6MHZSSqabsVxLCnJbgPdNhaahP/BaQFL7JwtQsW6EqDhctJEof6wQT9c5BzqKS0wM2EaURIE7pmoShRYAYYSYt/+kz+LdTXAn0cNXmFyGkg4CR/7HXOKthrnUbJyWAfLn8w8UY9vjMLQPoIH6zxky0ETcWE=
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    on:
+      branch: 1.1.0
```

Comment:
I believe this should be 1.0-stable. When we cut a 1.1-stable branch, the travis file on that branch should be updated accordingly.